server/routes/vendors.js

```diff
@@ -2,7 +2,13 @@ const express = require('express');
 const router = express.Router();
 const validator = require('validator');
 const Vendor = require('../db/Models/Vendor');
+const bcrypt = require('bcryptjs');
+const session = require('express-session');
```

Comment:
I think you only need `bcrypt` here, not `session`, `passport`, `LocalStrategy`, `redis` or `saltRounds`. Those all live in `server.js`